api/src/main/java/org/pmiops/workbench/api/OfflineRuntimeController.java

```diff
@@ -292,6 +298,38 @@ public ResponseEntity<Void> checkPersistentDisks() {
   private boolean notifyForUnusedDisk(LeonardoListPersistentDiskResponse disk, int daysUnused)
       throws MessagingException {
     String googleProject = leonardoMapper.toGoogleProject(disk.getCloudContext());
+    StringBuilder status = new StringBuilder();
+    try {
+      // If Label contains key aou-app-type it is either cromwell or Rstudio else its Jupyter
+      // runtime
+      if (((Map) disk.getLabels()).containsKey(LEONARDO_LABEL_APP_TYPE)) {
```

Comment:
isGce or App is AoU internal logic. If might worth to `[convert Leo response to our Disk object](https://github.com/all-of-us/workbench/blob/71d33734f678ae56442f28e32bee52447611fd91/api/src/main/java/org/pmiops/workbench/utils/mappers/LeonardoMapper.java#L132)`, and which already have isGceRuntime field. By doing that we have a central place to handle this logic instead of copying paste it to everywhere